browser/src/VariantList/filterVariants.ts

```diff
@@ -1,6 +1,51 @@
+import { Variant } from '../VariantPage/VariantPage'
 import { getCategoryFromConsequence } from '../vepConsequences'
 
-const filterVariants = (variants: any, filter: any, selectedColumns: any) => {
+type Categories = {
+  lof: boolean,
+  missense: boolean,
+  synonymous: boolean,
+  other: boolean,
+}
+
+export type VariantFilterState = {
+  includeCategories: Categories
+  includeFilteredVariants: boolean,
+  includeSNVs: boolean,
+  includeIndels: boolean,
+  includeExomes: boolean,
+  includeGenomes: boolean,
+  includeContext: boolean,
+  searchText: string,
+}
+
+export function getFilteredVariants(
+  filter: VariantFilterState,
+  variants: Variant[],
+  variantTableColumns: any
+){
+  const searchColumns = variantTableColumns.filter((column: any) => !!column.getSearchTerms)
```

Comment:
Another type comment, there are a couple `any`s here. If the `any` on line 27 gets narrowed down, the rest of those `any`s and variables in general in that chain should be able to be un-typed, since TypeScript will know what the type is at the beginning.